jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/JarCommandTest.java

```diff
@@ -315,4 +336,45 @@ public void testJar_baseImageSpecified()
     String output = new Command("docker", "run", "--rm", "cli-gcr-base").run();
     assertThat(output).isEqualTo("Hello World");
   }
+
+  public static void createJarFile(String name, String className, String classPath, String mainClass) throws IOException, URISyntaxException {
+    Path javaFilePath = Paths.get(Resources.getResource("jarTest/standard/" + className + ".java").toURI());
+    Path workingDir = Paths.get(Resources.getResource("jarTest/standard/").toURI());
+    System.out.println("Working dir: " + workingDir);
+
+    // compile the java file
+    JavaCompiler compiler = ToolProvider.getSystemJavaCompiler();
+    assertThat(compiler).isNotNull();
+    StandardJavaFileManager fileManager = compiler.getStandardFileManager(null, null, null);
+    Iterable<? extends JavaFileObject> compilationUnits = fileManager.getJavaFileObjectsFromFiles(
+        Collections.singletonList(javaFilePath.toFile()));
+    Iterable<String> options = Arrays.asList("-source", "1.8", "-target", "1.8");
+    JavaCompiler.CompilationTask task = compiler.getTask(null, fileManager, null, options, null, compilationUnits);
+    boolean success = task.call();
+    assertThat(success).isTrue();
```

Comment:
How about we use an if-else statement that proceeds with the jar building stage below if `task.call()` is true otherwise logs an error message  if the compilation fails?

Reply:
If we just silently log the message, then we might not notice the failure.

Reply:
Ah sorry, good point!

[Optional] Alternatively, to have a clear separation between the "arrange", "act" and "assert" blocks in our tests - consider using `Preconditions.checkState(task.call(), "<error message>");`.